dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipUdpConnLink.java

```diff
@@ -302,17 +300,17 @@ static SipUdpConnLink getPendingConnection() {
 		return current;
 	}
 
-	/**
-	 * establishes the outbound virtual connection
-	 */
-	private void connect(MessageContext messageContext) throws IOException {
-		String outboundChainName = m_channel.getOutboundChainName();
-		if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-			Tr.debug(this, tc, "<connect>", "outboundChainName = " + outboundChainName);
-		}
-		
-		// TODO connect async?
-	}
+    /**
+     * No Op -- Should already be connected and active. 
+     */
+    private void connect(MessageContext messageContext) throws IOException {
+        String outboundChainName = m_channel.getOutboundChainName();
+        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+            Boolean isActive =  ((SipUdpConnection)messageContext.getSipConnection()).getChannel().isActive();
+            Tr.debug(this, tc, "<connect>", "outboundChainName = " + outboundChainName + " isActive = " + isActive);
```

Comment:
For UDP's isActive boolean -- my understanding is that this is true when the socket is bound and ready to send/receive packets.